package.json

```diff
@@ -41,7 +41,7 @@
     "codeclimate-test-reporter": "^0.4.0",
     "colors": "^1.1.2",
     "css-loader": "^0.26.1",
-    "d2": "^26.1.7",
+    "d2": "^26.1.12",
```

Comment:
It is not a big deal but they are actually synced: https://github.com/dhis2/d2-ui/blob/v26/package.json#L3 && https://github.com/dhis2/d2-ui/blob/v26/package.json#L44 or am I missing sth?